recipes/igwn-segments/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "igwn-segments" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/i/igwn-segments/igwn_segments-{{ version }}.tar.gz
+  sha256: c1ea58c1d7656c99ae1d3a1db260a98f2f846ce7f0b22933050df7f6057492c4
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [python_impl != 'cpython']
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    # extras for cross-compiling:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+  host:
+    - pip
+    - python
+    - setuptools >=64
+    - setuptools-scm >=8
+  run:
+    - python
+
+test:
+  source_files:
+    - test/
+  requires:
+    - pip
+    - make  # [not win and not py>=313]
+    - python-lal  # [not win and not py>=313]
+  commands:
+    # check requirements
+    - python -m pip check
+    # run basic sanity test everywhere
+    - python -c "from igwn_segments import segment;
```

Comment:
Please move this into a run_test.py file.

Reply:
@ocefpaf, done in 245cf48.